akka-stream-tests/src/test/scala/akka/stream/scaladsl/FlowDelaySpec.scala

```diff
@@ -190,5 +190,14 @@ class FlowDelaySpec extends StreamSpec {
         .expectComplete()
     }
 
+    "not drop messages on overflow when EmitEarly" in {
+      val probe = Source(1 to 2)
+        .delay(50.millis, EmitEarly).withAttributes(Attributes.inputBuffer(1, 1))
```

Comment:
How likely does this test have a race condition? Can we increase this timeout a lot, to be sure that it still works when there's a 50 milli hiccup?

Reply:
Hello!

We reproduced it in our codebase with smaller delays (like 10ms) reliably so I personally don't know if more timeout will make it better. I suppose that a really simple flow with a source of two messages and only a delay should overflow before those 50ms. 

But I think that adding more delay would not make any harm, so I can increase it to 200ms for example just to ensure that we overflow.